linode/manager
# fix: [M3-8904] - Fix Storybook docgen

## Description 📝

Fixes docgen in Storybook, which is currently broken when launching through `yarn storybook`. Docgen converts TSDoc comments into documentation for props and components.

## Changes  🔄

- Removes the optimization from Storybook's `main.ts` and the related utility `getReactDocgenTSFileGlobs`, which was broken by #11184.
    - Others have [previously mentioned](https://github.com/linode/manager/pull/11229#discussion_r1838199423) that the benefits from this optimization may not justify the added complexity

## Preview 📷

| Before  | After   |
| ------- | ------- |
|  ![Screenshot 2024-11-14 at 3 50 41 PM](https://github.com/user-attachments/assets/14076b13-b964-4c83-9d62-e6f296de2db9)|  ![Screenshot 2024-11-14 at 3 48 54 PM](https://github.com/user-attachments/assets/564ae298-5b09-4325-9670-4c99587b0f7d)|

## How to test 🧪
1. Start Storybook via `yarn storybook`
2. Verify component stories have documentation generated from TSDoc comments (see screenshots).
